src/Parser/Rogue/Subtlety/Modules/Legendaries/SoulOfTheShadowblade.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Wrapper from 'common/Wrapper';
+
+class SoulOfTheShadowblade extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+	on_initialized(){
+		this.active = this.combatants.selected.hasFinger(ITEMS.SOUL_OF_THE_SHADOWBLADE.id);
+  }
+  
+	item() {
+		return {
+			item: ITEMS.SOUL_OF_THE_SHADOWBLADE,
+			result: <Wrapper>Equipped.</Wrapper>,
```

Comment:
Please add more information here aswell - if you want to help implementation of the other rogue builds, look in shared folder for hunters and implement soulring for all 3 in one go.